app/src/main/res/values/strings.xml

```diff
@@ -4,6 +4,9 @@
     <string name="title_mock_navigation">Mock Navigation</string>
     <string name="description_mock_navigation">Mock a navigation session using a mock location engine.</string>
 
+    <string name="title_component_hybrid_hav">Hybrid Navigation</string>
```

Comment:
NIT These `string` resources are not needed in the `app` module, we should remove.

Reply:
👍